examples/tf-sql-database/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "resource_group" {
+  description = "The name of the resource group in which to create the database."
+}
+
+variable "location" {
+  description = "The location/region where the database and server are created. Changing this forces a new resource to be created."
+  default     = "southcentralus"
+}
```

Comment:
since the previous example didn't include a default location would it make sense to remove this?